unumpy/linalg/__init__.py

```diff
@@ -1 +1,24 @@
 from ._multimethods import *
+
+__all__ = [
```

Comment:
Must be a list of strings, not a list of what's visible. E.g. the first element will be `"multi_dot"` vs `multi_dot`.